File: common/src/main/java/com/eliasnvx/krylix/core/HealthBarStyle.java
package com.eliasnvx.krylix.core;

import java.util.Locale;

public enum HealthBarStyle {
    BLOCKS,
    ASCII,
    DOTS,
    NUMBER_ONLY;

    @Override
    public String toString() {
        return "krylix.health_bar_style." + name().toLowerCase(Locale.ROOT);
    }
}

File: common/src/main/kotlin/com/eliasnvx/krylix/core/HealthBarStyle.kt
package com.eliasnvx.krylix.core

/**
 * Визуальный стиль полоски HP в индикаторе здоровья под прицелом (см. HealthIndicator).
 *
 * toString() переопределён на ключ перевода — Cloth Config берёт enum.toString() как
 * translation key для выпадающего списка (см. EnumListEntry), а голое имя константы
 * без этого не переводилось бы ни на один язык.
 */
enum class HealthBarStyle {
    BLOCKS,
    ASCII,
    DOTS,
    NUMBER_ONLY,
    ;

    override fun toString(): String = "krylix.health_bar_style.${name.lowercase()}"
}
